requirements.txt

```diff
@@ -9,7 +9,8 @@ cryptography==0.9.1
 debtcollector==0.5.0
 docker-py==1.3.1
 effect==0.1a13
-eliot==0.8.0
+eliot==0.9.0
+eliot-tree==15.3.0
```

Comment:
:question: Strictly speaking, the software doesn't require eliot-tree. It might be hard to make sense of logs without it, so I can see the motivation for including it here, but what's the policy? I'd have thought that `requirements.txt` should be exclusively for dependencies needed to run the software. Also arguably not much point installing it as part of Flocker's package unless something in our docs tells users what it is and how they use it.